pandas/core/generic.py

```diff
@@ -2328,6 +2328,11 @@ def to_json(
             'iso' = ISO8601. The default depends on the `orient`. For
             ``orient='table'``, the default is 'iso'. For all other orients,
             the default is 'epoch'.
+
+            .. deprecated:: 2.2.2
```

Comment:
I don't know if we are adding deprecations into the bugfix release for 2.2.2  @mroeschke

Reply:
We are not. Please add to `v3.0.0.rst`